src/commands/forging/info.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import BaseIPCCommand from '../../base_ipc';
+
+export default class InfoCommand extends BaseIPCCommand {
+	static description = 'Get forging information for the locally running node.';
+
+	static examples = [
+		'forging:info',
+	];
```

Comment:
```suggestion
	static examples = ['forging:info',];
```
I think this will fix the failing format test.

Reply:
maybe a good idea to also have example with flags `-d` & `--pretty`